manifests/h/HHDSoftware/FreeHexEditorNeo/7.50.02.8795/HHDSoftware.FreeHexEditorNeo.installer.yaml

```diff
@@ -0,0 +1,27 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: HHDSoftware.FreeHexEditorNeo
+PackageVersion: 7.50.02.8795
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+InstallerType: exe
+Scope: user
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: -silent
+  SilentWithProgress: -silent
+  Custom: -user
+UpgradeBehavior: install
+ReleaseDate: 2024-11-21
+Installers:
+- Architecture: x64
+  InstallerUrl: https://hhdsoftware.com/download/free-hex-editor-neo.exe
+  InstallerSha256: B8ACBDFF21CE28CDED12A4F3F5F829FB227AA07365817F7E845F08716DC017DE
+  InstallerSwitches: {}

```

Comment:
@russelbanks - Is this a Komac error?

https://github.com/PckgrMiranda/winget-pkgs/blob/HHDSoftware.FreeHexEditorNeo-7.50.02.8795-99762F06F864461389F1E452320322EF/manifests/h/HHDSoftware/FreeHexEditorNeo/6.44.01.6234/HHDSoftware.FreeHexEditorNeo.installer.yaml

Reply:
@PckgrMiranda - Not sure why this happened, but for some reason the switches didn't copy over from the previous version